### PR TITLE
run tests on gh

### PR DESCRIPTION
**Pull request by pumpurum2**
license field changed to pass yarn checks (now links to license file)

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ '**' ]
```
Comment (Eenae) [inline .github/workflows/node.js.yml:7]: also `dev`? no one wants their dev broken

Comment (pumpurum2) [reply .github/workflows/node.js.yml:7]: It was done since gh starts double run of action (one for push and one for pullrequst), will do at another way

Comment (Eenae) [reply .github/workflows/node.js.yml]: Now it's for pushes to any branch?

Comment (pumpurum2) [reply .github/workflows/node.js.yml]: yes, including pushes for pull requests (you can see 2 successful checks in this PR for the last commit)

**Review by Eenae — APPROVED** [review]

**Review by saninstein — APPROVED** [review]

**Pull request merged**